SerialColumn.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace yii2tech\csvgrid;
+
+/**
+ * Column is the base class of all [[CsvGrid]] column classes.
+ *
+ * @author Adegoke Obasa <adegokeobasa@gmail.com>
+ * @since 1.1.0
+ */
+class SerialColumn extends Column
+{
+    /**
+     * @inheritdoc
+     */
+    public $header = '#';
+
+
+    /**
+     * @inheritdoc
+     */
+    public function renderHeaderCellContent()
+    {
+        return parent::renderHeaderCellContent();
```

Comment:
This code is redundant: what the sense of redeclaring method just to call for its parent implementation?